spec/fe.test.js

```diff
@@ -78,3 +79,36 @@ describe('Quickview Body Properties', () => {
   });
 
 });
+
+describe('Quickview Details Properties', () => {
+  const wrapper = mount(
+     <QuickViewDetails details={data.quickview.details}/>
+   );
+
+  it('Should have the tour time', () => {
+    expect(wrapper.find('#time').text()).toEqual('1 hours');
+  });
+
+  it('Should have languages', () => {
+    expect(wrapper.find('#language').text()).toEqual('Languages Offered: in');
+  });
+
+  it('Should NOT have electronic vouchers', () => {
+    expect(wrapper.exists('#electronicVoucher')).toBe(false);
+  });
+
+  it('Should have paper vouchers', () => {
+    expect(wrapper.find('#paperVoucher').text()).toEqual('Paper Voucher Accepted');
+  });
+
+  it('Should NOT have instant confirmation', () => {
+    expect(wrapper.exists('#instantConfirmation')).toBe(false);
+  });
+
+  it('Should have cancellations', () => {
+    expect(wrapper.find('#cancellations').text()).toEqual('Free Cancellation up to 24 hours in advance');
+  });
+
+  // check for icons
+
+});
```

Comment:
Tests to consider adding might be:
- Simulating a click on a QuickView button and testing that the QuickView component renders; 
- Simulating a click on a different tab in the ToolBar component and testing the new view renders;
- Test that the number of experience components rendered is what you expect